repose-aggregator/core/repose-core/src/main/java/org/openrepose/powerfilter/PowerFilter.java

```diff
@@ -334,7 +334,10 @@ private PowerFilterChain getRequestFilterChain(HttpServletResponse httpResponse,
                 //Update the JMX bean with our status
                 configurationInformation.updateNodeStatus(clusterId, nodeId, false);
             } else {
-                requestFilterChain = new PowerFilterChain(filterChain, chain, router, metricsService);
+                requestFilterChain = new PowerFilterChain(filterChain, chain, router, metricsService,
+                        Optional.ofNullable(currentSystemModel.get().getReposeCluster().stream()
+                                .filter(cluster -> cluster.getId().equals(clusterId)).findFirst()
+                                .get().getFilters().getBypassUriRegex()));
```

Comment:
Yeah, i'm very quickly becoming not a fan of the token functional constructs they added in java 8.